scripts/alloc-APF-libs.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/sh
```

Comment:
Is this script used by anything?

Reply:
It's not used in our repo.  It's a convenience script to create the datasets used by the XMEM server in its PROC.  The comment header should explain this.

Reply:
This script isn't shipped as part of the product, so if it isn't used in here it's useless

Reply:
Right.  I propose I move it to `zowe-install-packaging` in `scripts/utils`.

Reply:
Yes - let's remove it from this PR and can have a separate one later to add it there

Reply:
Done.